service/routes.py

```diff
@@ -192,7 +217,11 @@ def update_a_product(user_id, product_id):
 ######################################################################
 @app.route("/shopcarts/<user_id>/items/<product_id>", methods=["DELETE"])
 def delete_a_product(user_id, product_id):
-    """Update a product in the shopcart"""
+    """Update a product in the shopcart
+    Args:
+        user_id (str): the user_id of the shopcart
+        product_id (str): the product_id of the product
+    """
```

Comment:
delete_a_product should have return value

Reply:
Your are right. Added the returns for delete